src/uci.cpp

```diff
@@ -96,10 +104,8 @@ void UCIEngine::loop() {
             sync_cout << "id name " << engine_info(true) << "\n"
                       << engine.get_options() << sync_endl;
 
-            sync_cout << engine.numa_config_information_as_string() << sync_endl;
-
-            if (auto str = engine.thread_binding_information_as_string(); !str.empty())
-                sync_cout << str << sync_endl;
+            print_numa_config_information();
+            print_thread_binding_information();
```

Comment:
This is exactly what I was talking about previously, we have to put these prints in random places because on_change is not called for the initial values. It would really simplify the code if we could do this, but it may be out of scope for this PR, needs a bit of thought to avoid issues.